lib/jiff-client.js

```diff
@@ -3477,9 +3477,9 @@
     jiff.share = function (secret, threshold, receivers_list, senders_list, Zp, share_id) {
       // type check to confirm the secret to be shared is a number
       // for fixed-point extension it should allow non-ints
-      if (typeof(secret) !== 'number' || Math.floor(secret) !== secret || secret < 0) {
+      /*if (typeof(secret) != "number" || Math.floor(secret) != secret || secret < 0) {
```

Comment:
Is there a reason for commenting out these checks?

Reply:
no, I think I needed it for testing but it should work negative number extension gets around this somehow which I believe it does now.